Payload/bleemsync/etc/bleemsync/FUNC/0010_execute.funcs

```diff
@@ -41,22 +41,21 @@ dump_fs_func(){
 		echo 1 > /sys/class/leds/red/brightness
 		ok=1
 		mediafs="$(df -T | grep -Ee ' /media$' | awk -e '{ print $2 }')"
-		[ ! -d "$dump_path/emmc_FS" ] && mkdir -p "$dump_path/emmc_FS"
+		mkdir -p "$dump_path/emmc_FS"
 		mount -o remount,ro /data && mount -o remount,ro /gaadata
-		if [ "$mediafs" == "vfat" ]; then
-			if ! dd if=/dev/mmcblk0 bs=1M | gzip | split -a3 -b4294967295 - "$dump_path/emmc_FS/gaadata.bin.gz."; then
+		if [ "$mediafs" = "vfat" ]; then
+			if ! dd if=/dev/mapper/gaadata bs=1M | gzip | split -a3 -b4294967295 - "$dump_path/emmc_FS/gaadata.bin.gz."; then
 				ok=0
 			fi
 		else
 			if ! dd if=/dev/mapper/gaadata bs=1M > "$dump_path/emmc_FS/gaadata.bin"; then
 				ok=0
 			fi
 		fi
-		if ! dd if=/dev/mmcblk0p10 bs=1M > "$dump_path/emmc_FS/data.bin"; then
+		if ! dd if=/dev/disk/by-partlabel/USRDATA bs=1M > "$dump_path/emmc_FS/data.bin"; then
 			ok=0
 		fi
-		rootdevice="$(mount | grep ' on / ' | awk -e '{ print $1 }')"
-		if ! dd "if=$rootdevice" bs=1M > "$dump_path/emmc_FS/root.bin"; then
+		if ! dd if=/dev/disk/by-partlabel/ROOTFS1 bs=1M > "$dump_path/emmc_FS/root.bin"; then
```

Comment:
Unless you're dumping recovery, which you've probably already tainted if you're running this code on there, we really don't need to check what the rootfs is and can hard code it in.